### PR TITLE
Hercules Studio product PR

### PR DESCRIPTION
**Pull request by dekel35**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Please confirm your submission meets all the criteria
+
```
Comment (hfiguiere) [inline .github/pull_request_template.md:2]: this file shouldn't be here

**Review by hfiguiere — CHANGES_REQUESTED** [review]
Can't even try to build right now.

**Comment by dekel35** [comment]
I believe I have addressed all issues in the first PR. Thanks!

**Comment by hfiguiere** [comment]
No you haven't.

**Comment by hfiguiere** [comment]
The manifest is still improperly named:

https://github.com/flathub/flathub/wiki/App-Requirements#repository-layout

File: HerculesStudio-manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- "--socket=wayland"
+- "--filesystem=host"
+- "--device=dri"
+modules:
+- name: Hercules-Studio
+  buildsystem: cmake
+  subdir: HerculesStudio
+  sources:
+  - type: git
+    url: file:///home/yakov/hercstudio-git-code
```
Comment (hfiguiere) [inline HerculesStudio-manifest.yaml:19]: still won't be able to find the source.

File: HerculesStudio-manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+app-id: com.jacobdekel.HerculesStudio
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+command: HerculesStudio
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
+- "--socket=wayland"
+- "--filesystem=host"
```
Comment (hfiguiere) [inline HerculesStudio-manifest.yaml:11]: as I said previously, this is a very broad filesystem permission.

File: HerculesStudio-manifest.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+app-id: com.jacobdekel.HerculesStudio
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+command: HerculesStudio
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
```
Comment (hfiguiere) [inline HerculesStudio-manifest.yaml:9]:
```suggestion
- "--socket=fallback-x11"
```
assuming this supports Wayland.

**Review by hfiguiere — CHANGES_REQUESTED** [review]
Changes that where previously requested.

File: HerculesStudio-manifest.yaml (hunk 2 above)
Comment (dekel35) [reply HerculesStudio-manifest.yaml]: Auch... committed the wrong file... SORRY!

File: HerculesStudio-manifest.yaml (hunk 3 above)
Comment (dekel35) [reply HerculesStudio-manifest.yaml]: Right. Sorry again 🙁

**Comment by dekel35** [comment]
OK. I appreciate if you can have one more look. Thanks!

**Comment by hfiguiere** [comment]
Still not fixed.

**Pull request closed without merging**